woocommerce-payments.php

```diff
@@ -9,7 +9,7 @@
  * Text Domain: woocommerce-payments
  * Domain Path: /languages
  * WC requires at least: 4.0
```

Comment:
@adrianduffell do we need to bump this to `5.4` or is the note in the `readme.txt` file enough?

Reply:
Sorry, please ignore my comment above, I just read the comment in the PR description

> Note the WC requires at least version remains at 4.0, as it is pending improvements in issue #1775 first.